.github/workflows/unit_test_mac.yml

```diff
@@ -0,0 +1,18 @@
+name: Unit test Mac
+
+on:
+  pull_request:
+
+jobs:
+  unit_test_mac:
```

Comment:
should mac test be in GH action?

Reply:
```
    runs-on: macos-latest
```
yes? as the go test is going to running on masos based github action agent.

Reply:
> should mac test be in GH action?

```
    runs-on: macos-latest
```
yes? as the go test is going to running on masos based github action agent.

Reply:
In may point of view, I hope we can decouple kepler analysis(how to process data) with get data from infrastructure(how to get data).
hence I hope by this PR, we allow Mac user can help with how to process data.
For example, health check or rest api logic.

Reply:
In may point of view, I hope we can decouple kepler analysis(how to process data) with get data from infrastructure(how to get data).
hence I hope by this PR, we allow Mac user can help with how to process data.
For example, health check or rest api logic.